schemas/ingest-events.v1.schema.json

```diff
@@ -0,0 +1,15 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "Error message from Relay",
+  "type": "object",
+  "properties": {
+    "type": { "type": "string" },
+    "payload": {
+      "description": "bytes"
```

Comment:
Is `"description": "bytes"` purely information for the human reader, or do we have a validator that special-cases this for msgpack messages?

Reply:
only for humans, this is not doing any validation